scripts/scraper.rb

```diff
@@ -0,0 +1,53 @@
+require 'nokogiri'
+require 'open-uri'
+require 'uri'
+
+# creates a scraper object to scrape all images within the given website
+class Scraper
```

Comment:
This is some 💯 Ruby.

![good job](https://media.giphy.com/media/P0RWkdsRpK7ss/giphy.gif)